test/frame_json_output.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+TMP="$(mktemp -d)"
+clean() { rm -rf "$TMP"; }
+trap clean EXIT
+
+cat << 'EOF' > "$TMP/schema.json"
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://example.com",
+  "$ref": "#/$defs/string",
+  "$defs": {
+    "string": { "type": "string" }
+  }
+}
+EOF
+
+# Test with --json flag
+"$1" frame "$TMP/schema.json" --json > "$TMP/result_json.txt"
+
+# Test with -j flag
+"$1" frame "$TMP/schema.json" -j > "$TMP/result.json"
+
+cat << 'EOF' > "$TMP/expected_json.txt"
+{
+  "frames": {
+    "https://example.com": {
+      "baseURI": "https://example.com",
+      "dialect": "https://json-schema.org/draft/2020-12/schema",
+      "pointer": "",
+      "relativePointer": "",
+      "schema": "https://example.com",
```

Comment:
Can you call these properties `root`, just to be consistent with the struct?

Reply:
For those whose pointer is empty(i.e, for first level) OR for all schema keywords: i.e,  `"schema": "https://example.com", `?